exercises/change/example.go

```diff
@@ -0,0 +1,191 @@
+// Package change demonstrates making change with fewest number of coins.
+package change
+
+import (
+	"errors"
+	"sort"
+)
+
+const testVersion = 1
+
+var (
+	ErrNoSolutionFound = errors.New("No solution")
+	ErrNegativeTarget  = errors.New("Negative Target")
+)
+
+// Change returns a list of coins in increasing value order to make
+// change for target amount from the coins given; an infinite supply of coins is assumed.
+func Change(coins []int, target int) (result []int, err error) {
```

Comment:
optional: it doesn't really look like the `result` and `error` names are used so it could just be `([]int, error)` and probably still be clear enough